docs/changelog.md

```diff
@@ -65,6 +65,11 @@ The default version of the distribution package does no longer contain the `janu
 This includes a packaged version of cassandra and elasticsearch. If you want to have `janusgraph.sh`, 
 you have to download distribution with the suffix `-full`.
 
+##### Gremlin server distributed with the release uses inmemory storage backend and no search backend
+
+Gremlin server distributed with the release uses inmemory storage backend and no search backend that is why 
```

Comment:
I think this section is a bit ambiguous as users could understand this as: _Gremlin Server now only works with inmemory_ whereas what you actually mean is that Gremlin Server is by default configured for the inmemory storage backend when started with `bin/gremlin-server.sh`.

Another nitpick here: It should always be _Gremlin **S**erver_.